platform/smallstep-agent.mdx

```diff
@@ -0,0 +1,263 @@
+---
+title: Smallstep Agent
+html_title: Install the Smallstep Agent on a Linux endpoint
+description: The Smallstep agent is used across security, IT, and DevOps teams to enable end-to-end network encryption, privileged access management, passwordless authentication, and much more.
+---
+
+While macOS and Windows can manage certificates and authentication settings via Mobile Device Management (MDM), Linux does not include automated remote management facilities. The Smallstep Agent brings vital certificate management features to your Linux users and endpoints. It is the [Smallstep App](./smallstep-app.mdx), without any GUI. It can be installed independently on any Linux device running systemd.
+
+Linux users should install either the Smallstep Agent or the Smallstep App&mdash;not both.
+
+In this document, we will install, configure, and start the Smallstep Agent on a Linux device running systemd. We also show how to use the agent’s built-in PKCS#11 (smart card) service. With the PKCS#11 service, you can access Smallstep certificates and keys from applications that support PKCS#11.
+
+# System Requirements
+
+- Supported operating systems:
+    - Enterprise Linux (RHEL, CentOS Stream, Rocky Linux, Alma Linux, etc)
+    - Ubuntu (Current Stable and LTS)
+    - Debian (Current Releases)
+    - Fedora (Current Releases)
+- We support `amd64` and `arm64` architectures
+- The following package dependencies will be installed:
+    - `polkitd`, to assign service manager privileges to the agent
+    - `p11-kit` and `tpm-tss2`, for TPM 2.0 integration
+- The following directories are used by default:
+    - runtime state in `/run/step-agent`
+    - configuration in `/etc/step-agent`
+    - certificates in`/var/lib/step-agent` and in your configured locations
+- The agent will connect to the following Smallstep hosts:
+    - Your CA:  `<your-team>.ca.smallstep.com` and subdomains
+    - Agent API: `control.infra.smallstep.com`
+    - Smallstep API: `gateway.smallstep.com`
+    - TPM Attestation CA: `att.smallstep.com`
+
+# Quick Install
+
+On a system with `bash` and `curl`, run:
```

Comment:
Maybe add:

run as the `root` user

so it is clear they need to be root to run the installer.